Packs/AccentureCTI/Integrations/ACTIIndicatorQuery/ACTIIndicatorQuery.py

```diff
@@ -334,24 +370,37 @@ def uuid_command(client: Client, args: dict, reliability: DBotScoreReliability,
             'LastSeen': str(last_seen_format)
         }
         analysis_info = _enrich_analysis_result_with_intelligence(analysis_info, doc_search_client)
+        ia, ir = [], []
+        alerts = analysis_info.get('Intelligence Alerts', [])
+        reports = analysis_info.get('Intelligence Reports', [])
+        if type(alerts) is dict:
+            alerts = list(analysis_info.get('Intelligence Alerts', []).values())
+            for alert in alerts:
+                ia.append(alert.split("/")[-1])
+        if type(reports) is dict:
+            reports = list(analysis_info.get('Intelligence Reports', []).values())
+            for report in reports:
+                ir.append(report.split("/")[-1])
+
+        context = {
+            "intelligence_alerts": ia,
+            "intelligence_reports": ir
+        }
```

Comment:
Seems like you are using this functionality quite often. Consider adding a method that implements it and call this method instead (not mandatory)